packages/styles/src/lib/Theme/components/MuiButton.ts

```diff
@@ -0,0 +1,96 @@
+import { Components } from '@mui/material';
+import { palette as themePalette } from '../palette';
+import { addTransparency } from '../util';
+
+const selectColor = (color?: string, isHover?: boolean): string | undefined => {
```

Comment:
It could be better to pass the theme here and get the color from it. If the engineer decides do change the primary color for example, we allow it. 
```suggestion
const selectColor = (color?: string, theme: Theme, isHover?: Boolean): string | undefined => {
```